topics/yoda.qmd

```diff
@@ -0,0 +1,35 @@
+---
+title: Yoda
+---
+Yoda is an application for institutions that supports RDM throughout the entire research cycle: from the safe and easy storage and sharing of data during the research process, to the sharing of data within research groups and projects and ultimately to research data archiving and publication.
+
+## Yoda and FAIR data
+Yoda helps the researcher make their data “[FAIR](./fair-principles.qmd)” by providing a solution that enables data discovery and sharing (i.e. findable, accessible). In addition, it facilitates and enforces the use of metadata, thereby, contributing to data interoperability and reusability. Yoda provides a platform for the implementation of standard workflows that can ensure metadata quality satisfies (institutional) policy requirements, for example, for data archiving and publication. In addition, Yoda’s integration with iRODS forms a single platform that accommodates both researchers with data heavy requirements, as well as those seeking an accessible, user-friendly solution.
+
+## Yoda for the researcher
+Yoda makes use of [iRODS](https://irods.org) as a foundation technology. However, it is much easier to use thereby making it more accessible to a wide range of researchers.
+
+Yoda presents researchers a comfortable, easy-to-use solution for the safe storage, sharing and organisation of research data that follows internationally accepted [FAIR](./fair-principles.qmd) data principles. Something increasingly demanded by both their institutions and funding organisations (such as the NWO).
+
+The following features make Yoda particularly attractive for researchers (end-users): 
```

Comment:
Should we add the advantage of being able to invite both VU-internal and external colleagues to a Yoda project space? I guess it's kind of implied in the final bullet point, but I think it's useful for researchers to have this explicit.